infrastructure/modules/backup.nix

```diff
@@ -45,7 +45,6 @@ with lib;
         User = "catcolab";
         ExecStart = getExe backupScript;
         Type = "oneshot";
-        EnvironmentFile = config.age.secrets.backendSecretsForCatcolab.path;
```

Comment:
Since we're running this as the `catcolab` user we don't need access to any DB credentials, the linux `catcolab` user is already set up to log in as the postgres `catcolab` user.